packages/postgres-database/src/simcore_postgres_database/migration/versions/3aa309471ff8_add_tag_to_groups_table_and_rm_tag_user_.py

```diff
@@ -99,6 +99,5 @@ def downgrade():
             """
         )
     )
-    op.alter_column("tags", "user_id", nullable=False)
```

Comment:
this looks pretty fishy to me. 
Why is this not a separate migration?

Reply:
This downgrade has data loss since the migration upgrade adds new table and columns. 
If the new table has group tags, there is no way to map a user back to the previous table and ``user_id`` is set to ``null``. We have two choices:
- keep the original non-nullable constraint and dropping all group tags
- drop the original non-nullable constraint and keeping all group tags

I have chosen the second. 
Nonetheless, this has no big impact since we do not use the downgrade mechanism.

Reply:
I finally got the point. This is a `DOWNGRADE`, my brain did not register it as such. 👍